molecule/default/create.docker.yml

```diff
@@ -45,3 +45,68 @@
           pgsql:
             image: "timescale/timescaledb:latest-pg13"
             env: { POSTGRES_PASSWORD: changeme }
+
+    - name: Create proxy to server version matches
+      set_fact:
+        p_to_s:
+          v60: 6.0
+          v64: 6.4
+          v70: 7.0
+
+    - name: Create database container for Zabbix server
+      community.docker.docker_container:
+        name: "{{ item.name }}-zserver-db"
+        image: postgres:13
+        state: started
+        recreate: true
+        networks:
+          - name: zabbix
+        env:
+          POSTGRES_DB: zabbix
+          POSTGRES_USER: "zabbix"
+          POSTGRES_PASSWORD: "zabbix"
+        links:
+          - "{{ item.name }}-zserver-db"
+      loop: "{{ molecule_yml.platforms }}"
+      when: "'zabbix_server_pgsql' in item.groups"
+
+    - name: Create container for Zabbix server
+      community.docker.docker_container:
+        name: "{{ item.name }}-zserver"
+        image: "zabbix/zabbix-server-pgsql:ubuntu-{{ p_to_s[item.zabbix_proxy_version] }}-latest"
+        state: started
+        recreate: true
+        networks:
+          - name: zabbix
+        env:
+          DB_SERVER_HOST: "{{ item.name }}-zserver-db"
+          POSTGRES_USER: "zabbix"
+          POSTGRES_PASSWORD: "zabbix"
+          POSTGRES_DB: "zabbix"
+        links:
+          - "{{ item.name }}-zserver-db"
+      loop: "{{ molecule_yml.platforms }}"
+      when: "'zabbix_server' in item.groups"
+
+    - name: Create container for Zabbix Web server
+      community.docker.docker_container:
+        name: "{{ item.name }}-zweb"
+        image: "zabbix/zabbix-web-nginx-pgsql:ubuntu-{{ p_to_s[item.zabbix_proxy_version] }}-latest"
+        state: started
+        recreate: true
+        published_ports:
+          - 8080:8080
+        networks:
+          - name: zabbix
+        env:
+          DB_SERVER_HOST: "{{ item.name }}-zserver-db"
+          POSTGRES_USER: "zabbix"
+          POSTGRES_PASSWORD: "zabbix"
+          POSTGRES_DB: "zabbix"
+          ZBX_SERVER_HOST: "{{ item.name }}-zserver"
+          PHP_TZ: "America/Toronto"
+        links:
+          - "{{ item.name }}-zserver-db"
+          - "{{ item.name }}-zserver"
+      loop: "{{ molecule_yml.platforms }}"
+      when: "'zabbix_server_web' in item.groups"
```

Comment:
This feels highly specific to testing the registering of zabbix_proxy via the api, and is probably better left to it's own `create.yml` file under `molecule/zabbix_proxy_psk/` right? (same goes for `destroy.yml`)

Reply:
We use this code for both zabbix_proxy and zabbix_proxy_psk

Reply:
Could we have these as separate create/destroy.yml in zabbix_proxy and source those from psk then?
It would simplify the logic with the groups somewhat. (i.e. we can just drop the when conditions for spinning up the official zabbix images.)

Reply:
Done with https://github.com/BGmot/community.zabbix/commit/83eac480d286d6d99108a3d24df2d0e5aa16e26a